Rovers/Rovers.tex

```diff
@@ -0,0 +1,309 @@
+This means being specific on how we are going to accomplish our goals.
+We want our team asking the question:"what material, what algorithm, with what method will we be accomplishing this?"
+It is very easy to over estimate the team's ability especially when the project is so new to the team.
+
+\subsection{Traceability}
+\label{subsec:traceability}
+
+It is important that the team members document the sources they use to gather information. 
+There will be a Google Drive folder that will hold notes with links to any books, articles, media or their sources that are relevant to the rover project. 
+The software and hardware will be tracked on GitHub. 
+This includes the technical report.
+
+\subsection{Accessibility}
+\label{subsec:plug-n-play}
+
+The project would be very ambitious. It would require at a minimum a handful (4-5) members each with different areas of expertise. 
+The members would need that knowledge as well as \LaTeX{}, notetaking, and good research practices. 
+This would be to deliver the MVP.
+A larger team would be able to accomplish more.  
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+
+This project would involve divvying out the different project areas to the different team members or small groups and each week meeting to discuss progress. 
+The team would take this information, refine it and put it into the technical report along with construction.  
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+
+The deliverable includes the technical report, Rover software, and Rover hardware.
+The primary deliverables of this project will be the technical report and the rover itself by the end of the second semester.
+It is also noted that the materials that the team will come across or notes should also be saved in an Google Drive folder for future reference. 
+The team will also present at SPEX design reviews and the weekly checkups at general meetings with the areas the team members are working on that week.
+The project is intended to last 2 semesters with the deliverable of a basic rover and a technical report.   
+
+\subsubsection{Technical Report}
+\label{techreport}
+
+The technical report of the rover is a design document complete with the hardware, software, and electrical components of the rover. This report is important as it allows for a future team to easily learn from the past team's experience. This Technical Report will be hosted on GitHub. This will also include the results of the rover tests and the teams recommendations  based on them. 
+
+\subsubsection{Rover Software}
+\label{roversoftware}
+The rover software has to be written and hosted on GitHub. This will include rover features such as navigation, control, and power regulation. 
```

Comment:
Describe some of the key pieces of software that will be required for a functional rover.